birdhouse/optional-components/scheduler-job-renew_letsencrypt_ssl_cert/config.yml.template

```diff
@@ -0,0 +1,15 @@
+- name: renew_letsencrypt_ssl
+  comment: Auto-renew LetsEncrypt SSL certificate
+  schedule: '${RENEW_LETSENCRYPT_SSL_SCHEDULE}'
+  command: '${COMPOSE_DIR}/deployment/certbotwrapper ${RENEW_LETSENCRYPT_SSL_EXTRA_OPTS}'
+  dockerargs: >-
+    --rm --name renew_letsencrypt_ssl
+    --volume /var/run/docker.sock:/var/run/docker.sock:ro
+    --volume ${COMPOSE_DIR}:${COMPOSE_DIR}:ro
+    --volume ${BIRDHOUSE_LOG_DIR}:/var/log/birdhouse:rw
+    --volume ${SCHEDULER_JOB_RENEW_SSL_CERT_SSL_CERT_DIR}:${SCHEDULER_JOB_RENEW_SSL_CERT_SSL_CERT_DIR}:rw
+    --env COMPOSE_DIR=${COMPOSE_DIR}
+    --env FORCE_CERTBOT_E2E=1
+    --env CERTBOT_RENEW=1
+    --env CERTBOTWRAPPER_LOGFILE=/var/log/birdhouse/renew_letsencrypt_ssl.log ${SCHEDULER_JOB_RENEW_SSL_CERT_LOCAL_ENV_DIR_VOLUME}
+  image: 'pavics/docker-compose-git:docker-18.09.7-compose-1.25.1'
```

Comment:
Please make this image configurable as well, same reason as the autodeploy, for compatibility with the other deprecate docker-compose v1 PR.

Also do not hard code `/var/log/birdhouse`.  I think we have a variable for that.

For consistency, please do a mass find and replace for that `/var/log/birdhouse`.

Reply:
/var/log/birdhouse is the location in the container. The location on the host is determined by the `BIRDHOUSE_LOG_DIR` which is used in the volume mount above.

Reply:
> Please make this image configurable as well, same reason as the autodeploy, for compatibility with the other deprecate docker-compose v1 PR.

done!